snippets/csharp/System.Linq/.editorconfig

```diff
@@ -0,0 +1,269 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
```

Comment:
I defer to the docs owners for this, but if editorconfig settings are specified shouldn't they be set repo-wide?

Reply:
This file should be removed from the PR. The coding style should match the settings from the editorconfig at the root of the repo.